src/render.ts

```diff
@@ -66,7 +67,7 @@ ${preview ? `\nopen({hash: ${JSON.stringify(resolvers.hash)}, eval: (body) => ev
     toc.show ? html`\n${renderToc(findHeaders(page), toc.label)}` : ""
   }
 <div id="observablehq-center">${renderHeader(options, page.data)}
-<main id="observablehq-main" class="observablehq">
+<main id="observablehq-main" class="observablehq${draft ? " observablehq-draft" : ""}">
```

Comment:
Since the “draft” state is a modifier rather than representing a unique element, it should have two hyphens.

```suggestion
<main id="observablehq-main" class="observablehq${draft ? " observablehq--draft" : ""}">
```